pootle/core/decorators.py

```diff
@@ -227,15 +230,17 @@ def set_project_resource(request, path_obj, dir_path, filename):
         resources = Store.objects.live().extra(
             where=[
                 'pootle_store_store.pootle_path LIKE %s',
+                'pootle_store_store.pootle_path ' + sql_regex + ' %s',
                 'pootle_store_store.pootle_path ' + sql_not_regex + ' %s',
-            ], params=[query_pootle_path, disabled_tps_regex]
+            ], params=[query_pootle_path, user_tps_regex, '^/templates/']
```

Comment:
Yes, ensuring `templates` is not in `user_tps` effectively ensures it is not in `user_tps_regex`. But I probably wasn't talking about doing it on Python but instead excluding it on the `user_tps` queryset.